### ISSUE TITLE
Nav a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1100]
- [ ] Landmarks must have a unique role or role/label/title (i.e. accessible name) combination

**Comment by m7kvqbe1** [comment]
This component is marked as deprecated - as such I have closed this ticket.
